.toolsharerc

```diff
@@ -1,5 +1,3 @@
 pinned_tool:
   - tool: "imp-ci"
     version: "20200206.175502.ed41e47ac7"
-  - tool: "imp-tool"
```

Comment:
Is this required? IMO, we should keep tools pinned if possible. We've had our CI break in the past because of issues in newer versions.

Reply:
The `imp-tool` specifically should no longer be pinned. It is baked into agents now. We are in the process of clearing this up by also removing it from toolshare itself to some degree.

Reply:
So would an `imp-tool` upgrade would require an agent re-bake?